telegram/__init__.py

```diff
@@ -175,7 +175,7 @@
 )
 
 
-from . import _version
+from . import _version, constants, error, helpers, request, warnings
```

Comment:
I don't remember if I deliberately didn't add this as an import, adding here since it is pep-8 compliant.

before this, importing those modules from `telegram` worked fine, `from telegram import *` also imported them

Reply:
and now `from telegram import *` no longer imports them? 🤔

Reply:
it does. I'm saying this change made no difference at all (except for shutting up Deepsource)

Reply:
ah, perfect :)